packages/angular-query-experimental/src/inject-mutation.ts

```diff
@@ -70,7 +70,11 @@ export function injectMutation<
         }
 
         effect(() => {
-          observer.setOptions(optionsFn(queryClient))
+          observer.setOptions(
+            runInInjectionContext(currentInjector, () =>
+              optionsFn(queryClient),
```

Comment:
Similar to the way `defaultQueryOptions` are applied to queries, should we apply `queryClient.defaultMutationOptions` here?

Reply:
Is this related to this PR's goal? Might be better to keep it separate.

Reply:
Somewhat - I think both should be merged, but this one isn't blocked & can go ahead first

Reply:
> Similar to the way `defaultQueryOptions` are applied to queries, should we apply `queryClient.defaultMutationOptions` here?

The other adapters don't do this either.